### DISCUSSION TITLE
Unable to install @scalar/nextjs-api-reference module on next js app router

### DISCUSSION THREAD

**Discussion by gallirohik** [scalar/scalar#1510] [Q&A] [answered]
Hi Team,
I am Trying to explore [nextjs-api-reference](https://github.com/scalar/scalar/blob/main/examples/nextjs-api-reference/package.json) example. However I am facing issues with module installations. 
 ```@scalar/api-client-react```
  ```@scalar/nextjs-api-reference```
Please help me to resolve this issue.
  Getting below error. :
  
```node
api-docs % npm install @scalar/nextjs-api-reference
npm install @scalar/nextjs-api-reference

npm ERR! code ETARGET
npm ERR! notarget No matching version found for @vueuse/core@^10.9.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

```node

api-docs % npm install @scalar/api-client-react
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @vueuse/core@^10.9.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

**Comment by gallirohik** [comment, marked as answer]
> Hi Team, I am Trying to explore [nextjs-api-reference](https://github.com/scalar/scalar/blob/main/examples/nextjs-api-reference/package.json) example. However I am facing issues with module installations. `@scalar/api-client-react` `@scalar/nextjs-api-reference` Please help me to resolve this issue. Getting below error. :
> 
> ```js
> api-docs % npm install @scalar/nextjs-api-reference
> npm install @scalar/nextjs-api-reference
> 
> npm ERR! code ETARGET
> npm ERR! notarget No matching version found for @vueuse/core@^10.9.0.
> npm ERR! notarget In most cases you or one of your dependencies are requesting
> npm ERR! notarget a package version that doesn't exist.
> ```
> 
> ```js
> api-docs % npm install @scalar/api-client-react
> npm ERR! code ETARGET
> npm ERR! notarget No matching version found for @vueuse/core@^10.9.0.
> npm ERR! notarget In most cases you or one of your dependencies are requesting
> npm ERR! notarget a package version that doesn't exist.
> ```

Found the issue, its because of npm cache.

`npm cache clean --force` helped :)

**Reply by amritk** [reply]
awesome, glad you sorted it out!
